index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Be careful about empty string at the end of each file

Reply:
Ok, i will!